apps/site-app/src/api/mock/accounts.ts

```diff
@@ -50,16 +50,16 @@ export function getDevWallets() : AccountMapState {
 
 export async function addDevLiveAccounts(accountsApi: IAccountStoreAPI) {
   // if dev:live we add 2 connected wallets.
-  addRemoteAccount('client1', accountsApi);
-  addRemoteAccount('client2', accountsApi);
+  addRemoteAccount('client1', accountsApi, true);
+  addRemoteAccount('client2', accountsApi, false);
 }
 
-async function addRemoteAccount(name: AccountName, accountsApi: IAccountStoreAPI) {
+async function addRemoteAccount(name: AccountName, accountsApi: IAccountStoreAPI, active: boolean) {
   const signer = await getSigner(name)
   const address = await signer.getAddress();
   const theSigner = signer as TheSigner;
   theSigner.address = address;
   theSigner._isSigner = true;
 
-  accountsApi.addAccount(name, theSigner, false, true);
+  accountsApi.addAccount(name, theSigner, false, active);
```

Comment:
Not sure what you mean by that - this just decides which one is active (same as switching accounts in the account switcher)

Reply:
This is actually mostly for NFT site, because only Client1 has a token, and it was annoying having to switch active account every time I wanted to test signing an NFT